rust/kernel/print.rs

```diff
@@ -12,6 +12,18 @@ use core::fmt;
 use crate::bindings;
 use crate::c_types::{c_char, c_void};
 
+pub use bindings::___ratelimit;
+pub use bindings::__printk_ratelimit;
+pub use bindings::ratelimit_state;
+
+extern "C" {
+    /// This function initializes ratelimit_state with DEFAULT_RATELIMIT_INTERVAL
+    /// and DEFAULT_RATELIMIT_BURST.
+    /// It should only be used inside [`print_macro_ratelimited`] macro.
```

Comment:
Maybe use `#[doc(hidden)]` to hide it from documentation and rust-analyzer autocompletion?

Reply:
ok